gpio_controller:
  ros__parameters:
    type: gpio_controllers/GpioCommandController
    gpios:
      - gpio
    command_interfaces:
      gpio:
        - interfaces:
           - "01"
    state_interfaces:
      gpio:
        - interfaces:
           - "01"
           - "02"